### ISSUE TITLE
Support the new JWT-VC 2.0 format

### ISSUE THREAD

**Issue by astanciu** [uport-project/veramo#1132]
There's a newly released native JWT VC representation spec, Version 2, here:
https://w3c.github.io/vc-jwt/#version-2

Some context available in this post here:
https://self-issued.info/?p=2316

Any plans to support this in the @veramo/credential-w3c ? Or is this a question better asked in the underly package, did-jwt-vc?

**Comment by mirceanis** [comment]
Yes, we do plan to adopt the new spec as well.

The work will be mostly in did-jwt-vc, but we'll also have to make sure that everything works here too

**Comment by astanciu** [comment]
Hi, curious if there's any updates on this front. Or curious how vermao plans to implement JWT2 support. Will there be a config option to use v1.1 or v2 or will the library switch to support only v2? Any estimated timeframes or other dependencies that you are waiting on?

Thanks

**Comment by mirceanis** [comment]
Yes, we will be adding support for v2. It's still too early to talk about that support now or how that would look like in practice.

**Comment by nickz-t3** [comment]
What is the state with this?

**Comment by mirceanis** [comment]
> What is the state with this?

wow, it's been a while.
We had a team volunteering to work on an implementation of VC2.0 in Veramo but so far there hasn't been any progress update.

If you'd like to raise a PR with an implementation, please say so :)
